README.md

```diff
@@ -122,8 +122,6 @@ If you are comfortable with using CMake, then a full clean, build, and test look
 By default, the C Media Driver is built as part of the C++ Build. However, it can be disabled via the CMake
 option `BUILD_AERON_DRIVER` being set to `OFF`.
 
-__Note__: C Media Driver is currently only supported on Mac and Linux.
```

Comment:
Changing the README and saying this will be supported on Windows is a bit early. And that is @mjpt777 and I to decide when that is ready.